Header header
float32 kRad
float32 omegaC

float32 kHeight
float32 kScaleA
float32 omegaA


float32 kScaleD
float32 kRate
float32 kOffset
float32 omegaD